src/guide/migration/global-api.md

```diff
@@ -106,6 +107,25 @@ In 3.0, the check of whether an element is a component or not has been moved to
 - This will be a new top-level option in the Vue CLI config.
   :::
 
+### `Vue.prototype` Replaced by `config.globalProperties`
+
+In Vue 2, `Vue.prototype` was commonly used to add properties that would be accessible in all components.
+
+The equivalent in Vue 3 is `config.globalProperties`. These properties will be copied across as part of instantiating a component within the application:
```

Comment:
*suggestion**: should we add a link to respective API reference chapter here, i.e. https://v3.vuejs.org/api/application-config.html#globalproperties ?

Reply:
Done.

I originally managed to persuade myself not to include a link but with hindsight my reasoning was dubious.